src/Rx/Weno/FacilityProperties.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Rx\Weno;
+
+class FacilityProperties
+{
+    public $facilityupdates;
+
+    public function __construct()
+    {
+        //do epic stuff!!
+    }
+
+    /**
+     * @return array
+     */
+    public function getFacilities()
+    {
+        $sql = "select id, name, street, city, weno_id from facility";
+        $list = sqlStatement($sql);
+        $facilities_list = [];
+        while ($row = sqlFetchArray($list)) {
+            $facilities_list[] = $row;
+        }
+        return $facilities_list;
+    }
+
+    /**
+     * @return
+     */
+    public function updateFacilityNumber()
+    {
+        $locations = $this->facilityupdates;
+        foreach ($locations as $location) {
+            sqlQuery("update facility set weno_id = ? where id = ?", [$location[1], $location[0]]);
+        }
+    }
+
+    /**
+     * @return
+     */
+    public function ifcolumexist()
+    {
+        $sql = "SELECT COLUMN_NAME FROM information_schema.COLUMNS " .
+            "WHERE " .
+            "TABLE_SCHEMA = '" . $GLOBALS['dbase'] . "'" .
+            "AND TABLE_NAME = 'facility' " .
+            "AND COLUMN_NAME = 'weno_id'";
+        $exist = sqlQuery($sql);
+        if (empty($exist)) {
+            sqlStatement("ALTER TABLE `facility` ADD `weno_id` VARCHAR(10) NULL DEFAULT NULL AFTER `oid`;");
```

Comment:
remove the AFTER part of this query (just opens up door for breaking the query in future if oid column ever goes away)

Reply:
actually, i would remove this entire function and just add the column per standard way (database.sql and upgrade sql script)

Reply:
done